README.md

```diff
@@ -33,7 +35,7 @@ To deploy MusicStore on Heroku, click the button below:
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
 
 ###NTLM authentication
-More information at src/MusicStore/StartupNtlmAuthentication.cs.
+More information at [src/MusicStore/StartupNtlmAuthentication.cs](https://github.com/aspnet/MusicStore/blob/dev/src/MusicStore/StartupNtlmAuthentication.cs).
```

Comment:
Can you make the links just be `[src/whatever.cs]` so that it'll work in any branch?